src/test/java/com/hillel/items_exchange/controller/LocationControllerIntegrationTest.java

```diff
@@ -182,4 +190,28 @@ void deleteLocations_shouldDeleteExistedLocation() throws Exception {
                 .andExpect(status().isOk())
                 .andReturn();
     }
-}
\ No newline at end of file
+
+    @WithMockUser(username = "admin", roles = {"ADMIN"})
+    @Test
+    @DataSet("database_init.yml")
+    void createLocationsInitFile_whenDataIsValid_shouldCreateFileAndReturnItsContent() throws Exception {
```

Comment:
@drw85 Current test throws `java.nio.file.NoSuchFileException: src\test\resources\LocationCities.txt` exception. Did you forget to add the file?

Reply:
Yeap). Fixed in next commit (will be done tomorrow)